src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -79,7 +79,18 @@ private int Run(string[] args)
                     }
                     else if (ArgumentMatch(arg, "type"))
                     {
-                        types.Add(value);
+                        if (value.Contains(";"))
```

Comment:
We don't need to special case `value.Contains(";") == true`. If value is not empty, we can just do `value.Split(';')`.